lib/screens/application.ex

```diff
@@ -10,10 +10,9 @@ defmodule Screens.Application do
 
     # List all child processes to be supervised
     children = [
-      Screens.Telemetry,
       {Screens.Cache.Owner, engine_module: Screens.Config.Cache.Engine},
       {Screens.Cache.Owner, engine_module: Screens.SignsUiConfig.Cache.Engine},
-      :hackney_pool.child_spec(:api_v3_pool, max_connections: 100),
+      :hackney_pool.child_spec(:api_v3_pool, []),
```

Comment:
I know `max_connections: 50` is the default, but I'd prefer it left in so that it's clear that it's intentional without looking through commit history for the line. 
Personal preference so I welcome disagreement

Reply:
Makes sense!